UPGRADE-4.3.md

```diff
@@ -145,6 +145,11 @@ Security
    }
    ```
 
+TwigBridge
+==========
+
+ * The `$requestStack` and `$requestContext` arguments of the `HttpFoundationExtension` have been deprecated. Pass a `Symfony\Component\HttpFoundation\UrlHelper` instance as the only argument instead.
```

Comment:
Can you change this sentence (and the other ones in the changelogs) to read like `deprecated the ...`?